composer.json

```diff
@@ -0,0 +1,90 @@
+{
+    "name": "swp/events-portal",
+    "license": "proprietary",
+    "type": "project",
+    "autoload": {
+        "psr-4": {
+            "SWP\\": "src/"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        },
+        "files": [
+            "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
+        ]
+    },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../web-publisher/src/SWP/Bundle/*"
```

Comment:
why is this needed? it is pointing to your local project. This shouldn't be here, the same with the below line.